readme.md

```diff
@@ -242,6 +242,7 @@
 - [lit-html](https://github.com/web-padawan/awesome-lit-html#readme) - HTML templating library for JavaScript.
 - [JAMstack](https://github.com/automata/awesome-jamstack#readme) - Modern web development architecture based on client-side JavaScript, reusable APIs, and prebuilt markup.
 - [Mobile Web Development](https://github.com/myshov/awesome-mobile-web-development#readme) - Creating a great mobile web experience.
+- [WYSIWYG](https://github.com/JefMari/awesome-wysiwyg#readme) - WYSIWYG editors
```

Comment:
The description here is not useful. It should describe what a WYSIWYG editor is in a succinct manner.